gooddata-java/src/test/java/com/gooddata/sdk/service/dataload/processes/ProcessServiceIT.java

```diff
@@ -330,8 +331,9 @@ public void shouldCreateProcessFromAppstorePackageRegistryMiss() throws Exceptio
              .respond()
                 .withStatus(200)
                 .withBody(readFromResource("/dataload/processes/appstoreProcess.json"));
-        DataloadProcess process = gd.getProcessService().createProcessFromAppstore(project, new DataloadProcess("appstoreProcess", ProcessType.RUBY.toString(),
-                "${PUBLIC_APPSTORE}:tag/prodigy-testing:/test/rubyHello")).get();
+        DataloadProcess newProcess = new DataloadProcess("appstoreProcess", ProcessType.RUBY.toString(),
+                "${PUBLIC_APPSTORE}:tag/prodigy-testing:/test/rubyHello");
+        DataloadProcess process = retry(() -> gd.getProcessService().createProcessFromAppstore(project, newProcess).get());
```

Comment:
These tests are running against mocked backends (using Jadler), so no retry is needed in this test class.

Reply:
@peter-plochan Done update code. Many Thanks.